theia-python-docker/README.md

```diff
@@ -3,31 +3,17 @@
 ### Build latest
 
 ```bash
-docker build . -t theiaide/theia-python:0.3.12
-docker build . -t theiaide/theia-python:latest
+docker build . -t theia-python:latest
```

Comment:
Should it me mentioned that it might be necessary to change the value of variable "version" in DockerFile to match the wanted version: latest or next?

Reply:
@marcdumais-work This is noted as well: `docker build --build-arg version=next . -t theiaide/theia-python:next`

Reply:
@jgroom33 I meant that specifying a version in the docker build command is not enough to actually have it use that version of Theia, when building the image. One also needs to specify ```latest``` or ```next``` in Dockerfile, variable ```version```.  But that's the case for other images too, so maybe no need to mention.